operator/validator/controller.go

```diff
@@ -976,6 +995,7 @@ func (c *controller) UpdateValidatorMetaDataLoop() {
 			if share.Liquidated {
 				return true
 			}
+
```

Comment:
we need to add here the same filter we do in the `startValidators` func. getting all the validators that in my subnets.

Reply:
we need to call the self subnets inside the loop to make sure we get the updated list of subnets for each iteration (to make sure its updated)

Reply:
please also check how much time "selfSubnets" function takes to make sure we are not slowing down this loop too much.

Reply:
> please also check how much time "selfSubnets" function takes to make sure we are not slowing down this loop too much.

300-900 µs

Reply:
sounds good. please add to the pr description